.pylintrc

```diff
@@ -82,7 +82,7 @@ required-attributes=
 bad-functions=map,filter,apply,input,file
 
 # Good variable names which should always be accepted, separated by a comma
-good-names=i,j,k,ex,Run,_,fd
+good-names=f,i,j,k,ex,Run,_,fd
```

Comment:
I only see one use case for that: `matches = [pattern.match(f) for f in files]` which should be rather translated to `matches = [pattern.match(something) for something in files]` (where `something` is `path` or `name`, depending on the intention) instead of introducing rule for the entire code base. You can also locally disable by `pylint: disable=invalid-name`.